### ISSUE TITLE
Markdown preview doesn't update for non-dendron managed markdown files

### ISSUE THREAD

**Issue by d1onys1us** [dendronhq/dendron#3750]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [ ] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [ ] Publish.  External Site Publish
- [x] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

If you open a regular markdown file in vs code which is not managed by dendron, live update does not work.

This is high priority because it means that I cannot at the moment recommend a Dendron installation for my team as it would break their preview for whatever other workflow they might have.

## Desktop (please complete the following information)

- Browser (if with published websites; include version): [e.g. chrome, safari]
- Versions (all that apply to bug):
  - Dendron Extension:
  - VS Code version:
  - Dendron CLI:

## Dendron Log file

Please attach the output of `> Dendron:Dev: Open Logs` here

**Comment by jonathanyeung** [comment]
@d1onys1us - Would an acceptable mitigation be to use regular VS Code preview for non-Dendron notes? 

If it's a non-dendron note, then the benefits of Dendron's preview largely wouldn't apply - i.e. note refs, backlinks, children, etc. 

@Dendron Dev - to fix this, instead of modifying existing preview logic in plugin/plugin-views, we should switch to using the render methodology in Dendron-Web where the HTML is generated on the extension side and sent to the web view.

**Comment by d1onys1us** [comment]
a quick update cc @jonathanyeung 

<img width="1382" alt="image" src="https://user-images.githubusercontent.com/13951458/199790251-dce142ff-2d09-409e-b0a4-c09db7ab86ef.png">

i noticed that in the vs code web view, i am also getting double preview buttons. in this repo, i have dendron inside of a folder `./handbook` and at the root, i have a non dendron managed file `./README.md`. i might change this later on, but this is the workflow which introduced a "bug". hope it helps :) thanks

**Comment by d1onys1us** [comment]
this might be outside of scope of this issue, if it is, let's open another issue. i can help with that, just let me know.

<img width="1374" alt="image" src="https://user-images.githubusercontent.com/13951458/199791016-157bd745-e555-4380-a0fa-d5a2f657c643.png">

i noticed that the theme is not carrying over with the dendron preview.

**Comment by d1onys1us** [comment]
fyi i have needed to uninstall dendron for time being because this is a blocker / breaks my other markdown files editing experience

**Comment by jonathanyeung** [comment]
@d1onys1us - you can still use VSCode's build-in preview markdown experience alongside Dendron's: 
- For VSCode's, you can run `Markdown: Open Preview to the side` command when you have a markdown note in focus. 
- For Dendron's, you can run `Dendron: Toggle Preview`, or use the icon. 

Was this the blocking aspect or was it something else? 

<img width="529" alt="image" src="https://user-images.githubusercontent.com/3203268/200453779-d53885d3-4bc4-4c40-b145-79199b49d99e.png">

**Comment by d1onys1us** [comment]
yes i can do that as well but it changes the user experience a bit, have to context switch between two markdown previews now

**Comment by d1onys1us** [comment]
hi @jonathanyeung i still think this is quite a blocker for me and it's degrading my DX. because i usually just click the native preview button, instead of using the command palette. so i have to either use command palette or uninstall dendron to get updates on preview to work properly.
